src/main/java/org/jabref/logic/util/StandardFileType.java

```diff
@@ -51,12 +54,13 @@
         return extensions;
     }
 
-    public static FileType newFileType(String... extensions) {
-        for (int i = 0; i < extensions.length; i++) {
-            if (extensions[i].contains(".")) {
-                extensions[i] = extensions[i].substring(extensions[i].indexOf('.') + 1);
-            }
-        }
-        return () -> Arrays.asList(extensions);
+    public static FileType fromExtensions(String... extensions) {
+        var exts = Arrays.asList(extensions);
+
+        return OptionalUtil.orElse(Arrays.stream(StandardFileType.values())
```

Comment:
I think `OptionalUtil.orElse` is no longer needed, as Java has now its own `orElse` method.

Reply:
@tobiasdiez  Yes, but it did not work with the type inference, it expected a type of StandardFileType in the orElse part and created a compile error when 
Therefore I had to use our util method:
https://stackoverflow.com/a/54238663

Please wait with merging I think I still need to convert the extensions to lower case before